config/economics.toml

```diff
@@ -0,0 +1,18 @@
+#Econimics config of the node
```

Comment:
Why do you need another economics.toml in this package?

Reply:
File was removed.